docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,460 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8fd79758-2771-41ce-b17e-e8bfdf5b1023",
+   "metadata": {},
+   "source": [
+    "# Get started with circuit cutting using gate cuts\n",
+    "\n",
+    "This guide demonstrates two working examples of gate cuts with the `qiskit-addon-cutting` package. It covers using gate cutting to reduce the circuit width (the number of qubits) and circuit depth (the number of circuit instructions). The gate cutting enables reconstructing a four-qubit circuit using two two-qubit subexperiments.\n",
+    "\n",
+    "The first example uses the [`efficient_su2`](/api/qiskit/qiskit.circuit.library.efficient_su2) ansatz and reconstructs the following observable:"
```

Comment:
The circuit creation code/description should be moved under the subsection "Gate cutting to reduce circuit width" if it only applies to that example.